bundles/org.openhab.binding.bluetooth.bluez/src/main/feature/feature.xml

```diff
@@ -4,7 +4,8 @@
 
 	<feature name="openhab-binding-bluetooth-bluez" description="Bluetooth Binding Bluez" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
-		<feature>openhab-transport-serial</feature>
+
+		<bundle dependency="true">mvn:com.github.hypfvieh/bluez-dbus-osgi/0.1.3</bundle>
```

Comment:
I have dbus-java working under OSGi with no troubles using following feature:

```
  <feature name="dbus" description="DBus transport" version="${project.version}">
    <feature prerequisite="true">wrap</feature>
    <bundle>mvn:com.github.jnr/jnr-unixsocket/0.33</bundle>
    <bundle>mvn:com.github.jnr/jnr-constants/0.9.15</bundle>
    <bundle>mvn:com.github.jnr/jnr-enxio/0.28</bundle>
    <bundle>mvn:com.github.jnr/jnr-ffi/2.1.15</bundle>
    <bundle>mvn:com.github.jnr/jnr-posix/3.0.58</bundle>
    <bundle>wrap:mvn:com.github.jnr/jnr-a64asm/1.0.0</bundle>
    <bundle>wrap:mvn:com.github.jnr/jnr-x86asm/1.0.2</bundle>
    <bundle>mvn:com.github.jnr/jffi/1.2.23/jar/complete</bundle>
    <bundle>wrap:mvn:com.github.jnr/jffi/1.2.23/jar/native</bundle>
    <bundle>wrap:mvn:com.github.hypfvieh/dbus-java/3.2.3</bundle>
    <bundle>wrap:mvn:com.github.hypfvieh/java-utils/1.0.6</bundle>
  </feature>
```
I discourage you from using `bluez-dbus-osgi` cause this packaging is really a fat jar which doesn't even make use of `Bundle-NativeCode` header. It doesn't have to, but it packs up all the jnr stuff making it pretty hard to run this with for example `NetworkManager` or `ModemManager`. Both use dbus as possible communication mechanism and give access to different parts related to network interfaces.
To try out with dbus stuff try adding this:
```
<bundle>wrap:mvn:com.github.hypfvieh/bluez-dbus/0.1.3</bundle>
```

Reply:
@J-N-K @kaikreuzer Is the @splatch's suggestion above acceptable?

Reply:
I agree that the bluez-dbus-osgi jar is a fat jar, which isn't really ideal - but it is always hard to tell, what can be left out from it.
It e.g. contains commons-lang3, which I do not see in your suggested feature definition, @splatch. Can you confirm that the bundles that you list there are enough as a replacement?
I cannot say much about the NetworkManager or ModemManager argument, since I am not deep enough in these details.
I'd think, we should be fine and the fat jar as a start (as it is confirmed to be working) and address the splitting up of it in a follow-up PR, if ok for everyone.

Reply:
@kaikreuzer I did test `dbus-java` for discovery of can interfaces through generated `NetworkManager` stub. It does work without commons. The `dbus-java` itself have very few dependencies (slf4j, jnr, java-utils), however `bluez-dbus` have some calls which introduce dependency to commons, see here: https://github.com/hypfvieh/bluez-dbus/search?q=org.apache.commons

Reply:
> I'd think, we should be fine and the fat jar as a start (as it is confirmed to be working) and address the splitting up of it in a follow-up PR, if ok for everyone.

Fine be me